fre/pp/ppyaml_test/pp.yaml

```diff
@@ -21,10 +21,10 @@ rose-suite:
     do_analysis: False
 
   directories:
-    history_dir: "/archive/c2b/am5/2022.01/c96L65_am5f1a0r0_amip/gfdl.ncrc5-intel22-classic-prod-openmp/history"
-    pp_dir: "/archive/$USER/canopy/am5/c96L65_amip/gfdl.ncrc5-deploy-prod-openmp/pp"
+    history_dir: "/archive/c2b/$(stem)/2022.01/$(expname)/$(platform)-$(target)/history"
+    pp_dir: "/archive/$USER/canopy/$(stem)/$(expname)/$(platform)-$(target)/pp" 
```

Comment:
In this case, "stem" wants to be "am5/2022.01". We can't do this, yet. So we could probably leave the hard-coded stem in the pp_dir:

pp_dir: "/archive/$USER/canopy/am5/2022.01/$(name)/$(platform)-$(target)/pp"